openpype/hosts/fusion/hooks/pre_fusion_setup.py

```diff
@@ -14,101 +15,36 @@ class FusionPrelaunch(PreLaunchHook):
-
-    def _sync_utility_scripts(self, env):
-        """ Synchronizing basic utlility scripts for resolve.
-
-        To be able to run scripts from inside `Fusion/Workspace/Scripts` menu
-        all scripts has to be accessible from defined folder.
-        """
-        if not env:
-            env = {k: v for k, v in os.environ.items()}
-
-        # initiate inputs
-        scripts = {}
-        us_env = env.get("FUSION_UTILITY_SCRIPTS_SOURCE_DIR")
-        us_dir = env.get("FUSION_UTILITY_SCRIPTS_DIR", "")
-        us_paths = [os.path.join(
-            os.path.dirname(os.path.abspath(openpype.hosts.fusion.__file__)),
-            "utility_scripts"
-        )]
-
-        # collect script dirs
-        if us_env:
-            self.log.info(f"Utility Scripts Env: `{us_env}`")
-            us_paths = us_env.split(
-                os.pathsep) + us_paths
-
-        # collect scripts from dirs
-        for path in us_paths:
-            scripts.update({path: os.listdir(path)})
-
-        self.log.info(f"Utility Scripts Dir: `{us_paths}`")
-        self.log.info(f"Utility Scripts: `{scripts}`")
-
-        # make sure no script file is in folder
-        if next((s for s in os.listdir(us_dir)), None):
-            for s in os.listdir(us_dir):
-                path = os.path.normpath(
-                    os.path.join(us_dir, s))
-                self.log.info(f"Removing `{path}`...")
-
-                # remove file or directory if not in our folders
-                if not os.path.isdir(path):
-                    os.remove(path)
-                else:
-                    shutil.rmtree(path)
+        self.log.info(f"Setting {py36_var}: '{py36_dir}'...")
+        self.launch_context.env[py36_var] = py36_dir
 
-        # copy scripts into Resolve's utility scripts dir
-        for d, sl in scripts.items():
-            # directory and scripts list
-            for s in sl:
-                # script in script list
-                src = os.path.normpath(os.path.join(d, s))
-                dst = os.path.normpath(os.path.join(us_dir, s))
+        # Add our Fusion Master Prefs which is the only way to customize
+        # Fusion to define where it can read custom scripts and tools from
+        self.log.info(f"Setting OPENPYPE_FUSION: {HOST_DIR}")
+        self.launch_context.env["OPENPYPE_FUSION"] = HOST_DIR
 
-                self.log.info(f"Copying `{src}` to `{dst}`...")
+        pref_var = "FUSION16_MasterPrefs"   # used by both Fu16 and Fu17
+        prefs = os.path.join(HOST_DIR, "deploy", "fusion_shared.prefs")
+        self.log.info(f"Setting {pref_var}: {prefs}")
+        self.launch_context.env[pref_var] = prefs
 
-                # copy file or directory from our folders to fusion's folder
-                if not os.path.isdir(src):
-                    shutil.copy2(src, dst)
-                else:
-                    shutil.copytree(src, dst)
+        __import__("avalon.fusion")
```

Comment:
Module `avalon.fusion` does not exists anymore and it does not make sense to import it in prelaunch hook. Import of `pyblish` also does not make sense here.